letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -714,9 +755,27 @@ def _copy_create_ssl_vhost_skeleton(self, avail_fp, ssl_fp):
             with open(avail_fp, "r") as orig_file:
                 with open(ssl_fp, "w") as new_file:
                     new_file.write("<IfModule mod_ssl.c>\n")
+                    sift = False
+
                     for line in orig_file:
-                        new_file.write(line)
+                        if self._sift_line(line):
+                            if not sift:
+                                new_file.write("# The following rewrite rules "
```

Comment:
nit: From this wording, I'd expect all disabled rewrite rules to be found directly under this comment. This is only the case, however, if all disabled rewrite rules are adjacent in the file.